### ISSUE TITLE
Please Provide a '-y' flag to non-interactively accept the comment

### ISSUE THREAD

**Issue by shariqmus** [Nutlope/aicommits#61]
Feature request:

Please Provide a '-y' flag to non-interactively accept the comment

To hack around I did the following:

File: /usr/local/lib/node_modules/aicommits/dist/cli.js

```

        // const confirmationMessage = yield inquirer_1.default.prompt([
        //     {
        //         name: 'useCommitMessage',
        //         message: 'Would you like to use this commit message? (Y / n)',
        //         choices: ['Y', 'y', 'n'],
        //         default: 'y',
        //     },
        // ]);
                
        //if (confirmationMessage.useCommitMessage === 'n') {
        if (0) {

```

Thank you.

**Comment by Nutlope** [comment]
Do you mean you want to auto-accept the commit message without seeing it?

**Comment by shariqmus** [comment]
> Do you mean you want to auto-accept the commit message without seeing it?

yes, in some repos, you are not too critical of the cimmit message but still need to provide meaningful ones, so it can serve that usecase.

Thanks!

**Comment by privatenumber** [comment]
Personally, I wouldn't rely on AI to this extent yet.

Let's keep this open for now and see if there's demand for this functionality.

**Comment by killgt** [comment]
I found my self doing the same hack. If i'm not happy with the result i can do git commit --amend and change the result.

**Comment by nthings** [comment]
https://github.com/Nutlope/aicommits/pull/75

**Comment by aslakhol** [comment]
This would be super nice for certain repos. I would prefer it if the commit message was echoed when commited with a non-interactive flag so that I can default to accepting the commit and just amend the ones that are not good enough.

**Comment by cdf1982** [comment]
+1 for the -y option. If you type it, you need to know what you're doing. But there are scenarios I can very much see its value :) Thanks for this amazing tool by the way :)
